### PR TITLE
faq: correct description about TiDB versions

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Close https://github.com/pingcap/docs/issues/6540

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11539
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
Hi, @morgo, I created this PR to address issue [#6540](https://github.com/pingcap/docs/issues/6540) you submitted. Could you please take a look and confirm whether this PR resolves the issue? Thanks.

**Comment by morgo** [comment]
> Hi, @morgo, I created this PR to address issue [#6540](https://github.com/pingcap/docs/issues/6540) you submitted. Could you please take a look and confirm whether this PR resolves the issue? Thanks.

LGTM

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -77,9 +71,9 @@ TiDB provides a few features and [tools](/ecosystem-tool-user-guide.md), with wh
 
 ### What's the difference between various TiDB master versions?
 
-The TiDB community is highly active. The engineers have been keeping optimizing and fixing bugs. Therefore, the TiDB version is updated quite fast. If you want to keep informed of the latest version, see [TiDB Release Timeline](/releases/release-timeline.md).
+The TiDB community is highly active. The engineers have been keeping optimizing features and fixing bugs. Therefore, the TiDB version is updated quite fast. If you want to keep informed of the latest version, see [TiDB Release Timeline](/releases/release-timeline.md).
 
-It is recommeneded to [deploy TiDB using TiUP](/production-deployment-using-tiup.md). TiDB has a unified management of the version number. You can view the version number using the following two methods:
+It is recommeneded to deploy TiDB [using TiUP](/production-deployment-using-tiup.md) or [using TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/stable) depending on your site conditions. TiDB has a unified management of the version number. You can view the version number using either of the following two methods:
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:76]:
```suggestion
It is recommeneded to deploy TiDB [using TiUP](/production-deployment-using-tiup.md) or [using TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/stable). TiDB has a unified management of the version number. You can view the version number using one of the following methods:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**